faucet/exec/Main.hs

```diff
@@ -2,42 +2,29 @@
 
 module Main where
 
-import Control.Lens (views)
 import Fmt ((+|), (|+))
 import Loot.Base.HasLens (lensOf)
-import Loot.Log (logError, logInfo)
+import Loot.Log (logInfo)
 import Options.Applicative (execParser, fullDesc, helper, info, progDesc)
 
 import Dscp.CommonCLI (versionOption)
 import Dscp.Config (buildConfig, configParamsParser)
 import Dscp.Core
 import Dscp.Faucet
 import Dscp.Resource.Keys
-import Dscp.Witness.Web
 
 main :: IO ()
 main = do
     (faucetParams, faucetConfig) <- getFaucetParams
     launchFaucetRealMode faucetConfig faucetParams $ do
-        printSourceInfo faucetParams
+        printSourceInfo
         serveFaucetAPIReal (_fpWebParams faucetParams)
   where
-    printSourceInfo params = do
-        wc <- views (lensOf @WitnessClient) (hoistWitnessClient liftIO)
+    printSourceInfo = do
         pk <- view $ lensOf @(KeyResources FaucetApp) . krPublicKey
         let addr = mkAddr pk
-
         logInfo $ "Faucet source address: " +| addr |+ ""
 
-        let DryRun dryRun = _fpDryRun params
```

Comment:
shouldn't you nuke `_fpDryRun` now too?

Reply:
It was used primarily to test frontend making transactions without witness backend, not only to suppress this check at start.
Taking this into account, does it worth removing this flag anyway?

Reply:
If it's used somewhere else, then probably it's not.
Otherwise, if it's not used anywhere, then there's no reason to keep it around at all.

Reply:
Ok, there is one usage site in tx making logic, thus I remain.